File: 0229-majority-element-ii/0229-majority-element-ii.go
func majorityElement(nums []int) []int {
    count := make(map[int]int)
    res := []int{}

    for i := 0; i < len(nums); i++ {
        count[nums[i]]++
    }

    for k, v := range count {
        if v > len(nums)/3 {
            res = append(res, k)
        }
    }

    return res   
}

File: 0229-majority-element-ii/0229-majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        count = {}
        res = []
        for n in nums:
            count[n] = 1 + count.get(n, 0)
        
        for n, freq in count.items():
            if freq > len(nums)/3:
                res.append(n)
        
        return res
